brave/brave-core
# Move Brave Wallet into its own settings section

Resolves https://github.com/brave/brave-browser/issues/14069
Resolves https://github.com/brave/brave-browser/issues/14062

This follows similar work to the IPFS settings section, but for wallet updates.
- It fixes a bug from IPFS settings work where not all settings menu items were being included.
- It also fixes the ordering of the items to match the settings (before IPFS was after appearance even though it was listed after extensions in the meu).
- It also removes a duplicate IPFS icon.
- Finally it does a little cleanup on the proxy files.


<img width="1016" alt="Screen Shot 2021-02-09 at 11 43 14 AM" src="https://user-images.githubusercontent.com/831718/107396711-02eea900-6acc-11eb-9d84-8706e2a9f076.png">


## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Make sure the settings sections are back from https://github.com/brave/brave-browser/issues/14062
- Make sure the crypto wallets settings still work
- Make sure things are ordered correclty
